cloud/amazon/ecs_service.py

```diff
@@ -346,35 +441,41 @@ def main():
             if not module.check_mode:
                 if module.params['load_balancers'] is None:
                     loadBalancers = []
```

Comment:
This isn't being used, it should be more like [this](https://github.com/ansible/ansible-modules-extras/compare/devel...zacblazic:ecs-service-params#diff-411a22dc031b2d8f51d4b69e307537e5R434).